subjects/apps/solution.js

```diff
@@ -141,6 +147,9 @@ const Chat = React.createClass({
         <div ref="messages" className="messages" onScroll={this.handleScroll}>
           <MessageList auth={auth} messages={messages}/>
         </div>
+        <div ref="newNotifier" className="new-notifier">
```

Comment:
If we had another piece of state, possibly something like `hasNewMessages`, then we could just decide here whether or not to include this `<div>` in the `render` output.